app/assets/stylesheets/components/_sub_header.scss

```diff
@@ -33,3 +33,15 @@
 #all_trips {
   display: none;
 }
+
+#active_families {
+  display: block;
+}
+
+#not_active_families {
+  display: none;
+}
+
+#all_families {
+  display: none;
+}
```

Comment:
Thought about suggesting you alphabetise these and the functions above but then I thought they are probably grouped logically from how they connect.

Reply:
Yeah, Elle said that the classes don't need to be in order also bc sometimes they need to have a certain order so I grouped by logic! 😉